### ISSUE TITLE
The Preview README.md for local fine tuning is not the repo, has steps missing

### ISSUE THREAD

**Issue by sirredbeard** [microsoft/vscode-ai-toolkit#94]
When performing local fine tuning, this Preview README.md appears:

![image](https://github.com/user-attachments/assets/57c63842-5271-462f-97c8-e4416b3cc32b)

I cannot find the text of this file in the repository, unlike the others for remote fine tuning, so I cannot open a PR to suggest fixes.

However, it is containing missing steps.

1. The AI toolkit base is activating the `base` conda environment, not the one specified in the project setup page.
2. It says to simply run `conda activate [conda-env-name]` but you need to run `conda env create -f setup/conda-environment.yml` first, otherwise conda doesn't see the environment specified in the project setup page.

**Comment by sirredbeard** [comment]
The file also contains a typo in:

```
To just try try the base model without fine-tuning you can run this command after activating conda.
```

**Comment by sirredbeard** [comment]
This appears to be done in `first_time_setup.sh` but is not executing, or needs to be documented that the user should run.

**Comment by swatDong** [comment]
For local finetuning, AI Toolkit will download the model raw files and execute `first_time_setup.sh` when generating the project. E.g.,
<img width="683" alt="image" src="https://github.com/user-attachments/assets/a351435f-0c4e-4859-988e-b2e556d6f03f">

Do you see any similar output from the log panel, or any error message?

BTW, good suggestion on the doc and I'll update the document in case user needs to run that manually.

**Comment by sirredbeard** [comment]
@swatDong I did not see that on first run, but I set up another fine tuning project, it downloaded the models, and attempted to re-run the `first_time_setup.sh`. Re-running the script failed then because I already had a conda env with the same name, but it was simple enough to pick it up from there. I am not sure what happened on the first run, may have been user error.

Thank you for updating the doc.
